docs/quality-assurance/bdd.md

```diff
@@ -0,0 +1,80 @@
+# Behavioural-Driven Design (BDD) at UKHO
```

Comment:
BDD stands for Behaviour-driven development?
```suggestion
# Behaviour-Driven Development (BDD) at UKHO
```

Reply:
Quite right!